library/src/main/java/com/patloew/colocation/CoLocation.kt

```diff
@@ -60,6 +59,22 @@ interface CoLocation {
     @RequiresPermission(anyOf = [Manifest.permission.ACCESS_COARSE_LOCATION, Manifest.permission.ACCESS_FINE_LOCATION])
     suspend fun isLocationAvailable(): Boolean
 
+    /**
+     * Returns a single location fix on the device.
+     *
+     * Unlike [getLastLocation], this method could cause active location computation on the device.
```

Comment:
Please adapt the docs to be the same as the [`FusedLocationProvider.getCurrentLocation()` docs](https://developers.google.com/android/reference/com/google/android/gms/location/FusedLocationProviderClient#getCurrentLocation(int,%20com.google.android.gms.tasks.CancellationToken)).

Reply:
Updated to match the `FusedLocationProviderClient` docs! I left off the last section, since we're hiding the `CancellationToken` and handling that directly.